src/app/state/actions.tsx

```diff
@@ -1586,7 +1586,7 @@ export const assignBoard = (board: GameboardDTO, groups: Item<number>[] = [], du
     }
 
     if (scheduledStartDate != undefined) {
-        scheduledStartDate?.setUTCHours(0, 0, 0, 0);
+        // Unline with the due date, we want to preserve the hour assigned at the UI level, unless we want to move that logic here.
```

Comment:
You might want to check that this logic of making sure the scheduled start date is still in the future checks out still. 
To be honest, I think the logic to set the hour to 7am should be in this file because other logic here (the two comparisons against `scheduledStartDate`) could be affected.
```suggestion
        scheduledStartDate?.setUTCHours(7, 0, 0, 0);
```